domains/vishwa.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "VishwaiOSDev",
+        "email": "vishwar3001@gmail.com"
+    },
+    "records": {
+        "CNAME": "https://vishwaiosdev.github.io"
```

CNAME doesnt allow any trailing slashes (/)

```suggestion
        "CNAME": "vishwaiosdev.github.io"
```